static/js/components/FilterCandidateList.jsx

```diff
@@ -46,20 +53,66 @@ const useStyles = makeStyles(() => ({
       marginLeft: "0.5rem",
     },
   },
+  formRow: {
+    margin: "1rem 0",
+  },
+  redshiftField: {
+    display: "inline-block",
+    marginRight: "0.5rem",
+  },
 }));
 
+function getStyles(classification, selectedClassifications, theme) {
+  return {
+    fontWeight:
+      selectedClassifications.indexOf(classification) === -1
+        ? theme.typography.fontWeightRegular
+        : theme.typography.fontWeightMedium,
+  };
+}
+
 const FilterCandidateList = ({
   userAccessibleGroups,
   setQueryInProgress,
   setFilterGroups,
   numPerPage,
 }) => {
   const classes = useStyles();
+  const theme = useTheme();
 
   const defaultStartDate = new Date();
   defaultStartDate.setDate(defaultStartDate.getDate() - 1);
   const defaultEndDate = new Date();
 
+  const ITEM_HEIGHT = 48;
+  const MenuProps = {
+    PaperProps: {
+      style: {
+        maxHeight: ITEM_HEIGHT * 4.5,
+        width: 250,
+      },
+    },
+  };
+
+  // Get unique classification names, in alphabetical order
+  const { taxonomyList } = useSelector((state) => state.taxonomies);
+  const latestTaxonomyList = taxonomyList.filter((t) => t.isLatest);
+  let classifications = [];
+  latestTaxonomyList.forEach((taxonomy) => {
+    const currentClasses = allowedClasses(taxonomy.hierarchy).map(
+      (option) => option.class
+    );
+    classifications = classifications.concat(currentClasses);
+  });
+  classifications = classifications
+    .filter(
+      (classification, index) =>
+        classifications.indexOf(classification) === index
+    )
```

Comment:
Is this filter expression actually doing anything, or am I just not seeing something? It looks like it just always reproduces the original list.

Reply:
It's there to take out the duplicates in the list (some classifications have the same name but just different hierarchies)

Reply:
Ah, okay, how about `Array.from(new Set(classifications))`, then?